tests/providers/cncf/kubernetes/utils/test_pod_manager.py

```diff
@@ -252,6 +253,26 @@ def test_parse_log_line(self):
         assert timestamp == pendulum.parse(real_timestamp)
         assert line == log_message
 
+    @mock.patch("airflow.providers.cncf.kubernetes.utils.pod_manager.PodManager.read_pod_logs")
+    def test_fetch_container_logs_returning_last_timestamp(self, mock_read_pod_logs):
+        timestamp_string = "2020-10-08T14:16:17.793417674Z"
+
+        mock_read_pod_logs.return_value = [bytes(f"{timestamp_string} message", "utf-8"), b"notimestamp"]
+
+        self.first_time = True
```

Comment:
This looks a bit clumsy to define here self variable that is used only in this test.
I can suggest this approach:
```
def mock_container_is_running(...):
    if mock_container_is_running.first_time:
        ...
    ...

mock_container_is_running.first_time = True
```